credentialdigger/models/snippet_model/snippet_model.py

```diff
@@ -133,17 +141,32 @@ def _pre_process(self, raw_data):
 
         >>> raw_data = '"password": "#####", "!@AAA12")'
         >>> print(self._pre_process(raw_data))
-        ['password', 'AAA12']
+        ['password','"#####', '!@AAA12']
+
+        Manual execution
+        --------
+        >>> raw_data = 'pwd = "pwd_123";'
+        >>> print(self._pre_process(raw_data))
+        [
+            words = [pwd, pwd_123]
+            strings = [pwd_123] # Will not be converted into camel_case
+            camel_case_words = [Pwd, pwd_123]
+        ]
```

Comment:
The `_pre_process` method doesn't return this list. If you want to give a good description of these steps, you can do it in the notes (more on this in the final comment of the review)